### ISSUE TITLE
How to install

### ISSUE THREAD

**Issue by 546pvp** [Compewter/VEAL#8]
Maybe i am just blind, But How to install and use it correctly?

**Comment by compewter** [comment]
Nope, not blind. It would have been helpful if I added instructions to the readme.

I havent run this for many years so YMMV.

The following should work:
`git clone {this repo}`
`cd whoof`
`npm install`
`npm start`

if it gives you an error about concurrently not being a command:
`npm install -g concurrently`
